rmw_dps_cpp/src/type_support_common.hpp

```diff
@@ -67,8 +68,15 @@ _create_type_name(
     RMW_SET_ERROR_MSG("members handle is null");
     return "";
   }
-  return
-    std::string(members->package_name_) + "::" + sep + "::dps_::" + members->message_name_ + "_";
+
+  std::ostringstream ss;
+  std::string message_namespace(members->message_namespace_);
+  std::string message_name(members->message_name_);
+  if (!message_namespace.empty()) {
+    ss << message_namespace << "::";
+  }
+  ss << "dds_::" << message_name << "_";
```

Comment:
Can you change this to sep << "::dps_" ...?

Reply:
I changed `dds_` into `dps_`.
However, the separator is not necessary anymore (in the last commit I also removed it from the arguments).

See here for the field `message_namespace`
https://github.com/ros2/rosidl/commit/176bbe2640983e74fd1104017a9e153f089eef21

Reply:
Great, thanks for the clarification